front/src/components/UI/ActionInput/UpdateProduct.tsx

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import {
+  ButtonStyled,
+  FormStyled,
+  LabelStyled,
+  InputStyled,
+} from "./AuctionInput.styled";
+import { isEmpty } from "../../App/SalesArea/SalesArea";
+import { useState } from "react";
+
+const UpdateProduct = () => {
+  const [formInputsValidity, setFormInputsValidity] = useState({
+    name: true,
+    id: true,
+  });
+
+  const updateProductHandler = (event: any) => {
+    event.preventDefault();
+
+    const enteredId = event.target.elements.id.value;
+    const enteredName = event.target.elements.name.value;
+
+    const enteredIdIsValid = !isEmpty(enteredId);
+    const enteredNameIsValid = !isEmpty(enteredName);
+
+    setFormInputsValidity({
+      name: enteredNameIsValid,
+      id: enteredIdIsValid,
+    });
+
+    if (enteredIdIsValid && enteredNameIsValid) {
+      axios.put("http://localhost:4000/products/" + enteredId, {
```

Comment:
co jesli zmieni się numer portu api? Dużo plików trzeba będzie przeszukać i pozmieniać. Adres bazowy zwykle trzymany jest w pliku .env, ale żeby nie obarczać cię tym, to można chociaż wynieść to do zwykłego consta w jakimś głównym pliku utils czy coś